### PR TITLE
Create 11-11-16 -- Markets were offline.md

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: incidents/11-11-16 -- Markets were offline.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Summary
+
+Date: ~Fri 11 Nov 2016 (~1324) to Fri 11 Nov 2016 (~1452) (~1.5 hours)
+
+
+## What happened?
+
+Images which had particularly long urls would fail to load.
```
Comment (alicebartlett) [inline incidents/11-11-16 -- Markets were offline.md]: **that http://www.quickanddirtytips.com/education/grammar/which-versus-that-0

File: incidents/11-11-16 -- Markets were offline.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Thur 3 Nov 2016
+
+~1215: [Changes in `next-article`](https://github.com/Financial-Times/next-article/pull/1665) were deployed which swapped all images in articles from the old image service to the new image service.
+
+1324 : Mustafa Sogancilar noticed that the market images were missing from the site and announced in `ft-next-support`.
+
+1332: Origami team ([Jake Champion](https://github.com/JakeChampion)) [contacted via Slack](https://financialtimes.slack.com/archives/ft-next-support/p1478698330005277).
+
+~1337: [Changes in `next-article`](https://github.com/Financial-Times/next-article/commit/a45a109116bd817b584648558e26bf3c03a5885d) were merged by [Jake Champion](https://github.com/JakeChampion) that reverts to using the original image service.
```
Comment (alicebartlett) [inline incidents/11-11-16 -- Markets were offline.md]: Found this sentence a little tricky to parse, suggest this reword:

+~1337: [Jake Champion](https://github.com/JakeChampion) reverts [changes in `next-article`](https://github.com/Financial-Times/next-article/commit/a45a109116bd817b584648558e26bf3c03a5885d) so it no longer uses the (broken) v2 image service.

File: incidents/11-11-16 -- Markets were offline.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+1413: [Changes in `next-article`](https://github.com/Financial-Times/next-article/pull/1666) were merged.
+
+1416: [Changes in `n-content-transform`](https://github.com/Financial-Times/n-content-transform/pull/28) were merged.
+
+~1452: Changes went live and issue was confirmed as being resolved.
+
+
+## Cause
+
+Changes made to `n-content-transform` and `next-article` to use the new image service surfaced a bug in our new image service which fails to load images from long urls.
```
Comment (alicebartlett) [inline incidents/11-11-16 -- Markets were offline.md]: * from urls longer than [how many] characters

File: incidents/11-11-16 -- Markets were offline.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## What could be done to avoid this in future?
+
+- Document the current breaking feature and place it in our migration guide
```
Comment (alicebartlett) [inline incidents/11-11-16 -- Markets were offline.md]: add link to issue for this for continued tracking

File: incidents/11-11-16 -- Markets were offline.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- Write integration test which makes request for images at varying url lengths.
```
Comment (alicebartlett) [inline incidents/11-11-16 -- Markets were offline.md]: Add link to issue or pr

**Comment by JakeChampion** [comment]
@alicebartlett comments have been addressed, please review

**Pull request merged**